src/test/kotlin/com/dprint/services/editorservice/EditorServiceFormatterTest.kt

```diff
@@ -0,0 +1,261 @@
+package com.dprint.services.editorservice
+
+import com.dprint.config.ProjectConfiguration
+import com.dprint.messages.DprintAction
+import com.dprint.utils.warnLogWithConsole
+import com.intellij.openapi.components.service
+import com.intellij.openapi.project.Project
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.shouldBe
+import io.mockk.clearAllMocks
+import io.mockk.coEvery
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.mockkObject
+import io.mockk.mockkStatic
+import io.mockk.runs
+import io.mockk.verify
+import kotlinx.coroutines.runBlocking
+
+class EditorServiceFormatterTest : FunSpec({
```

Comment:
In general this test is testing a lot of behavior when the editorService is null. This shouldn't really happen so we should either simplify it so we can test behavior in a single test and ensure that no jobs get triggered and appropriate logging is happening.